en/debug/failure_injection.md

```diff
@@ -0,0 +1,49 @@
+# System Failure Injection
+
+System failure injection allows you to cause different types of sensor and system failures using the [MAVLink shell](../debug/mavlink_shell.md#mavlink-shell) or other PX4 console.
```

Comment:
Or MAVSDK failure plugin: https://mavsdk.mavlink.io/main/en/cpp/api_reference/classmavsdk_1_1_failure.html

Reply:
Thanks. that makes quite a bit of difference :-). Fixed in https://github.com/PX4/PX4-user_guide/pull/1354